app/javascript/dashboard/App.vue

```diff
@@ -88,6 +94,8 @@ export default {
     },
     async initializeAccount() {
       await this.$store.dispatch('accounts/get');
+
+      this.$store.dispatch('accounts/limits');
```

Comment:
Shall we move this call to the upgrade banner and show the banner only if it is Chatwoot Cloud?